### PR TITLE
adding alt props for html scripts

### PR DESCRIPTION
**Pull request by ericblancas23**
## Related Issue

Fixes #

## Checklist:

- [X ] I have read the [contributor's guide](https://wildaid.github.io/contribute/index.html).
- [X] I linked an issue in the previous section
- [X ] I have commented on the linked issue
- [x] I was assigned the linked issue (not required)
- [X ] I have tested the change to the best of my ability against the [sandbox](https://wildaid.github.io/contribute/sandbox.html) or a [local build](https://wildaid.github.io/build).
- [X ] All my images have [appropriate alt tags](https://wildaid.github.io/style/2020/10/02/Alt-Text.html)

Optional items:

- [ ] My change adds new text and requires a change to translations.
- [ ] I was not able to test... (explain below, e.g. you did not have permissions to test a specific feature)
- [ ] This change depends O-FISH Realm repository changes (explain below)
- [ ] This change depends O-FISH Web repository changes (explain below)

* **Optional: Add any explanations here** 
All requested items have been provided alt props for the html tags 

* **Optional: Add any relevant screenshots here**

### CODE REVIEW

File: _posts/build_fdn/2020-06-09-Setup-Commandline-for-Realm-Import.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ site: build
 The commandline interface to import the Realm app code is called "realm-cli". We will set this up in this step, then import all the Realm code in the next step.
 
 1. realm-cli setup
-   1. Install realm-cli - follow the instructions for manual install or `npm` install at <A HREF="https://docs.mongodb.com/realm/deploy/realm-cli-reference/" target="_blank">https://docs.mongodb.com/realm/deploy/realm-cli-reference</A><BR><BR>
+   1. Install realm-cli - follow the instructions for manual install or `npm` install at <A alt="cli install link" HREF="https://docs.mongodb.com/realm/deploy/realm-cli-reference/" target="_blank">https://docs.mongodb.com/realm/deploy/realm-cli-reference</A><BR><BR>
```
Comment (Sheeri) [inline _posts/build_fdn/2020-06-09-Setup-Commandline-for-Realm-Import.md:12]: Hi! regular links - <A> tags - do not need alt tags. Only images do, thanks! Please remove this change. Everything else looks good, will merge when the alt tag on this link is removed.

Comment (ericblancas23) [reply _posts/build_fdn/2020-06-09-Setup-Commandline-for-Realm-Import.md]: @Sheeri thanks!, alt tag has been removed, should be good to go. Thanks!

**Review by Sheeri — APPROVED** [review]
Looks good to me!

**Comment by Sheeri** [comment]
Congratulations! 🎉 🙌 🎆 ㊗️ 

This PR is great and has been merged into the code. Thanks so much for contributing! If you'd like, you can request to work on another issue. We really appreciate your effort!

**Comment by Sheeri** [comment]
Hi @ericblancas23! 
We noticed you haven't gotten your MongoDB digital swag. To claim your O-FISH badge, head on over to the [MongoDB Community forums](https://developer.mongodb.com/community/forums/t/open-source-contributors-pick-up-your-badges-here/10167/9), login, and reply - making sure to link to this PR.

**Pull request merged**